conf/report/testing.py

```diff
@@ -0,0 +1,74 @@
+from selenium import webdriver
+from selenium.webdriver.support.ui import Select
+import sys
+
+def setDriver(driver_location, driver=None):
```

Comment:
`driver_location` has to be optional - in most cases it doesn't need to be provided to webdriver's constructor.
Alternatively you can leave only `driver=None` argument and use it for providing the driver object, so that `setDriver()` will set default one (Chromium), `setDriver(webdriver.Firefox())` will use Firefox, etc.